### PR TITLE
Add zIndex property for changing default zIndex property

### PR DESCRIPTION
**Pull request by LKay**
Resolves issue #13.

### CODE REVIEW

File: src/sidebar.js [hunk 1]
```diff
@@ -200,9 +197,9 @@ class Sidebar extends React.Component {
   }
 
   render() {
-    let sidebarStyle = styles.sidebar,
+    let sidebarStyle = Object.assign({}, styles.sidebar, { zIndex: this.props.zIndex + 1 }),
```
Comment (balloob) [inline src/sidebar.js]: Please use `update(orig, {$merge: {…merge objects…}}` syntax. Object.assign is not available in all browsers.

I'm down to polyfill `Object.assign` but only if all calls to `update()` get replaced. I don't want to mix two approaches.

Comment (LKay) [inline src/sidebar.js]: Ok, I'll change it to use `update()` later. Actually I was surprised you use this approach and mix es6 with es5 features. You're using babel to compile code anyway and it includes its polyfills. There are many places where the code could be written more es6 (7?) way.

Comment (balloob) [inline src/sidebar.js]: I was under the impression that the babel polyfill is not included by default. You have to explicitly require it.

I would probably do a better job if I would write it today but when I started ES6 was all still pretty new to me. You're welcome to help improve it :+1:

Comment (LKay) [inline src/sidebar.js]: I will try to refactor the code a little during my free time. Anyway good work so far! :+1:

**Comment by Kerumen** [comment]
:+1:

**Comment by balloob** [comment]
@LKay, do you have plans to finish this PR or shall I close it?

**Pull request closed without merging**